src/start_up/mod.rs

```diff
@@ -102,7 +102,7 @@ pub fn prepare_clock(block0: &Block) -> Result<Clock, Error> {
 
     let initial_epoch = ClockEpochConfiguration {
         slot_duration,
-        slots_per_epoch: slots_per_epoch.unwrap_or(10 * 10),
+        slots_per_epoch: slots_per_epoch.unwrap_or(100), // FIXME
```

Comment:
whats wrong with the default value?

Reply:
This should actually be `date::EPOCH_DURATION`, but rust-cardano doesn't export that constant at the moment. (Also it's not a very good default probably.)

Reply:
Hm, I see the default got removed from rust-cardano, so maybe we should just panic here if `slots_per_epoch` is not set? @vincenthz

Reply:
yes, slots_per_epoch & slot_duration should always be present now